frontend/src/pages/add-quiz-page.tsx

```diff
@@ -0,0 +1,138 @@
+import { Dispatch, SetStateAction, useState } from "react";
+import { Link } from "react-router-dom";
+
+type Question = {
+    id: number,
+    word: string,
+    options: string[],
+    type: string
+}
+
+type Quiz = {
+    name: string,
+    description: string,
+    questions: Question[]
+}
+
+const QuestionSentence: React.FC<{ type: string, word: string }> = ({ type, word }) => {
```

Comment:
can we format this method to make it more readable?

Reply:
Done.